### PR TITLE
Fix directory table ci test unstable.

### PR DESCRIPTION
**Pull request by wenchaozhang-123**
As two directory schema tables have the approximately the same in tuples count, the join plan is unstable. We fix this by adding more tuples to one schema table.

Authored-by: Zhang Wenchao zwcpostgres@gmail.com

Fixes #ISSUE_Number

### Type of Change
- [ ] Bug fix (non-breaking change)
- [ ] New feature (non-breaking change)
- [ ] Breaking change (fix or feature with breaking changes)
- [ ] Documentation update

### Test Plan

- [ ] Unit tests added/updated
- [ ] Integration tests added/updated
- [ ] Passed `make installcheck`
- [ ] Passed `make -C src/test installcheck-cbdb-parallel`

### Impact

**Performance:**

**User-facing changes:**

**Dependencies:**

### Checklist
- [ ] Followed [contribution guide](https://cloudberry.apache.org/contribute/code)
- [ ] Added/updated documentation
- [ ] Reviewed code for security implications
- [ ] Requested review from [cloudberry committers](https://github.com/orgs/apache/teams/cloudberry-committers)

### Additional Context

⚠️ **To skip CI:** Add `[skip ci]` to your PR title. Only use when necessary! ⚠️

---

### CODE REVIEW

File: src/test/regress/output/directory_table.source [hunk 1]
```diff
@@ -802,23 +802,75 @@ NOTICE:  dir_table1 INSERT AFTER ROW  (seg2 127.0.1.1:7004 pid=31192)
 ERROR:  syntax error at or near "WITH"
 LINE 1: ...dir_table1 FROM STDIN 'nation5' WITH TAG 'nation' WITH TAG '...
                                                              ^
+\COPY BINARY dir_table1 FROM '@abs_srcdir@/data/nation.csv' 'nation6';
+NOTICE:  dir_table1 INSERT AFTER ROW  (seg0 127.0.1.1:7002 pid=21790)
```
Comment (gfphoenix78) [inline src/test/regress/output/directory_table.source:806]: Does these seg info like ip:port, pid count when matching the result?

Comment (wenchaozhang-123) [reply src/test/regress/output/directory_table.source:806]: Yes, seg info will be ignore in regression diffs.

**Review by gfphoenix78 — APPROVED** [review]

**Comment by avamingli** [comment]
> As two directory schema tables have the approximately the same in tuples count, the join plan is unstable. 

Will Analyze make plan stable?

**Comment by wenchaozhang-123** [comment]
> > As two directory schema tables have the approximately the same in tuples count, the join plan is unstable.
> 
> Will Analyze make plan stable?

Actually not, has added analyze table before executing explain in test case, join plan is still unstable.

**Comment by avamingli** [comment]
> > > As two directory schema tables have the approximately the same in tuples count, the join plan is unstable.
> 
> > 
> 
> > Will Analyze make plan stable?
> 
> 
> 
> Actually not, have added analyze table before executing explain in test case, join plan is still unstable.

Is it a problem of analyze on directory tables?
The plan should be stable after ANALYZE like normal tables, planer only cares about costs.

**Comment by wenchaozhang-123** [comment]
> > > > As two directory schema tables have the approximately the same in tuples count, the join plan is unstable.
> > 
> > 
> > > 
> > 
> > 
> > > Will Analyze make plan stable?
> > 
> > 
> > Actually not, have added analyze table before executing explain in test case, join plan is still unstable.
> 
> Is it a problem of analyze on directory tables? The plan should be stable after ANALYZE like normal tables, planer only cares about costs.

Analyze directory tables means analyze directory tables' schema tables which are just heap tables which is not different at all. What's more, have not changed any codes of directory table in planner. So it's should not be a problem of analyze directory table. I guess maybe the two tables have the same count tuples cause the join plan not stable. Sometimes a join b, othertimes b join a.

**Review by my-ship-it — APPROVED** [review]

**Pull request merged**